include/picongpu/CMakeLists.txt

```diff
@@ -378,6 +378,21 @@ else(ISAAC_FOUND)
     endif()
 endif(ISAAC_FOUND)
 
+################################################################################
+# PIConGPU Workarounds
+################################################################################
+
+set(PIC_COMPUTE_CURRENT_THREAD_LIMITER_DEFAULT OFF)
+if(ALPAKA_ACC_GPU_HIP_ENABLE)
+    set(PIC_COMPUTE_CURRENT_THREAD_LIMITER_DEFAULT ON)
+endif()
+option(PIC_COMPUTE_CURRENT_THREAD_LIMITER "Compute current results with HIP alpaka backend are wrong when more threads than particles in a frame will be used (possible compiler BUG)."
+    "ON means the number of threads will be limited to number of particles in a frame." ${PIC_COMPUTE_CURRENT_THREAD_LIMITER_DEFAULT})
```

Comment:
I believe CI failed because cmake only supports `\` for strings on multiple lines (like C++ macro), but not closing and opening of "" like C++ strings